src/frontend/magnify/src/components/design-system/Formik/Input/FormikInput.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
```

Comment:
In previous commit, you have created a similar file then you remove it now. Please clean your commits.